transport/http2_server.go

```diff
@@ -845,3 +972,14 @@ func (t *http2Server) RemoteAddr() net.Addr {
 func (t *http2Server) Drain() {
 	t.controlBuf.put(&goAway{})
 }
+
+func getJitter(v time.Duration) time.Duration {
```

Comment:
That's actualy a good idea and looked into it turns time.Add works on type time.Time while MaxConnectionAge is type time.Duration (type int64).